scene/gui/rich_text_label.cpp

```diff
@@ -5114,13 +5114,17 @@ void RichTextLabel::set_text(const String &p_bbcode) {
 	if (text == p_bbcode) {
 		return;
 	}
-
 	text = p_bbcode;
+	_apply_translated_bbcode(text);
+}
+
+void RichTextLabel::_apply_translated_bbcode(const String &p_bbcode) {
+	String xl_text = atr(p_bbcode);
```

Comment:
The input is not necessarily BBCode, but is always `text`.

```suggestion
void RichTextLabel::_apply_translation() {
	String xl_text = atr(text);
```

Reply:
Good point, much simpler this way, thanks. Commit updated.